evalml/model_understanding/graphs.py

```diff
@@ -534,21 +580,43 @@ def graph_partial_dependence(pipeline, X, feature, class_label=None, grid_resolu
         # Don't specify share_xaxis and share_yaxis so that we get tickmarks in each subplot
         fig = _subplots.make_subplots(rows=rows, cols=cols, subplot_titles=class_labels)
         for i, label in enumerate(class_labels):
-
             # Plotly trace indexing begins at 1 so we add 1 to i
```

Comment:
nit-pick: This comment applies to the `col=(i % 2) + 1` expression so I think we move it closer to that or get rid of it

Reply:
Thanks!!! Good call.  I must have a bunch of stuff missing in here but I definitely have PR fatigue on this one.